hw/ip_templates/rv_plic/rtl/rv_plic.sv.tpl

```diff
@@ -238,7 +238,7 @@ module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
     .hw2reg,
 
     // SEC_CM: BUS.INTEGRITY
-    .intg_err_o(alerts[0]),
+    .intg_err_o(alerts[i]),
```

Comment:
This one should remain `alerts[0]` as it's a specific alert outside the `gen_alert_tx` generate loop

Reply:
Thanks!